homeassistant/components/pandora/media_player.py

```diff
@@ -275,26 +282,30 @@ def _update_song_position(self):
             self._attr_state = MediaPlayerState.PAUSED
         self._time_remaining = time_remaining
 
-    def _log_match(self):
+    def _log_match(self) -> None:
         """Log grabbed values from console."""
+        assert self._pianobar is not None
         _LOGGER.debug(
             "Before: %s\nMatch: %s\nAfter: %s",
             repr(self._pianobar.before),
             repr(self._pianobar.match),
             repr(self._pianobar.after),
         )
 
-    def _send_pianobar_command(self, service_cmd):
+    def _send_pianobar_command(self, service_cmd: str) -> None:
         """Send a command to Pianobar."""
+        assert self._pianobar is not None
         command = CMD_MAP.get(service_cmd)
         _LOGGER.debug("Sending pinaobar command %s for %s", command, service_cmd)
         if command is None:
             _LOGGER.warning("Command %s not supported yet", service_cmd)
+            return
```

Comment:
If the `command` is not supported, it shouldn't be send to the `_pianobar` entity.